examples/controlpanel/next.config.js

```diff
@@ -1,10 +1,10 @@
-require('dotenv').config();
+require('dotenv-defaults').config({ debug: process.env.DEBUG });
```

Comment:
please favor dotenv-extended instead of dotenv + dotenv-defaults because we used that package in other projects too: https://github.com/keithmorris/node-dotenv-extended